### ISSUE TITLE
CMake building issues

### ISSUE THREAD

**Issue by hvonw** [Chocobo1/qbittorent_msys2#5]
Hi @Chocobo1.

Tried to build with cmake but it fails. 
Is it because of recent changes? Can you help me out with this?
Thanks.

```
/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: ../base/libqbt_base.a(session.cpp.obj): in function `convertIfaceNameToGuid':
qBittorrent/src/base/bittorrent/session.cpp:4548: undefined reference to `ConvertInterfaceNameToLuidW'
/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: qBittorrent/src/base/bittorrent/session.cpp:4551: undefined reference to `ConvertInterfaceLuidToGuid'
collect2.exe: error: ld returned 1 exit status
make[2]: *** [src/app/CMakeFiles/qBittorrent.dir/build.make:1311: src/app/qbittorrent.exe] Error 1
make[1]: *** [CMakeFiles/Makefile2:130: src/app/CMakeFiles/qBittorrent.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```
------------------------------------------------------------------------------------------------------------
https://github.com/msys2/MSYS2-packages/issues/1681 (Couldn't built with qmake because of this bug.)
https://github.com/qbittorrent/qBittorrent/pull/11293/commits/7276a79cefe00f3e82fec8760ec876aa8a276df0

**Comment by Chocobo1** [comment]
This should fix it: https://github.com/qbittorrent/qBittorrent/pull/11382

However cmake is not officially supported in qbittorrent, so I'm closing this.

**Comment by Chocobo1** [comment]
>This should fix it: qbittorrent/qBittorrent#11382

If not, you could try adding `Netio` to `link_libraries()` list.
